src/common/containers/build-details.js

```diff
@@ -26,6 +27,9 @@ class BuildDetails extends Component {
         { this.props.isFetching &&
           <span>Loading...</span>
         }
+        { this.props.error &&
+          <Message status='error'>{ this.props.error.message || this.props.error }</Message>
```

Comment:
Ah yes, it's a tricky one.

Currently in the store error can be `false` or error object. That's why I needed to add `PropTypes.bool` to make both valid values. Not sure if I can limit it to object and `false` only.

But I just realised using `null` instead of `false` would make it much less confusing.